### PR TITLE
Cy 2528 change aws region to input

### PR DESCRIPTION
**Pull request by AdarShaked**

### CODE REVIEW

File: aws-example-network/README.md [hunk 1]
```diff
@@ -33,8 +33,6 @@ You must have these secrets on your Cloudify Manager `tenant`:
 
   * `aws_access_key_id`: Your AWS credentials access key ID.
   * `aws_secret_access_key`: Your AWS credentials access key secret.
-  * `aws_region_name`: The EC2 region, for example `us-east-1`.
-  * `ec2_region_endpoint`: The EC2 region service endpoint, for example `ec2.us-east-1.amazonaws.com`.
   * `aws_availability_zone`: Your preferred availability zone in your EC2 region, such as `us-east-1c`.
```
Comment (EarthmanT) [inline aws-example-network/README.md]: Zone should also not be a secret. Make it input as well.

File: hello-world-example/README.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ cfy secrets create openstack_auth_url --secret-string <value>
 For **AWS**:
 
 ```shell
-cfy install aws.yaml -i aws_region_name=eu-central-1
```
Comment (EarthmanT) [inline hello-world-example/README.md]: Maybe provide both examples. One as a way to use default and one as a way to override.

File: prometheus/aws.yaml [hunk 3]
```diff
@@ -21,6 +21,7 @@ inputs:
 
   region_name:
```
Comment (EarthmanT) [inline prometheus/aws.yaml]: lets rename to `aws_region_name` for consistency.

File: virtual-machine/README.md [hunk 4]
```diff
@@ -150,7 +150,6 @@ If you are an AWS user, you must create the following secrets:
 
   - [ ] `aws_access_key_id` AWS Access Key, e.g.: `cfy secrets create -u aws_access_key_id -s ...................`.
   - [ ] `aws_secret_access_key`: AWS Secret Access Key, e.g.: `cfy secrets create -u aws_secret_access_key -s ...................`.
-  - [ ] `aws_region_name`: AWS region name Key, e.g.: `cfy secrets create -u aws_region_name -s us-east-1`.
   - [ ] `ec2_region_endpoint`: AWS region endpoint, e.g.: `cfy secrets create -u ec2_region_endpoint -s ec2.us-east-1.amazonaws.com`.
   - [ ] `aws_availability_zone` AWS availability zone, e.g.: `cfy secrets create -u aws_availability_zone -s eu-central-1b`.
```
Comment (EarthmanT) [inline virtual-machine/README.md]: also remove the ec2_region_endpoint and aws_availability_zone

File: virtual-machine/aws.yaml [hunk 5]
```diff
@@ -34,12 +34,16 @@ inputs:
       The username of the agent running on the instance created from the image.
     default: 'centos'
 
+  aws_region_name:
+    type: string
+    default: 'ca-central-1'
+
```
Comment (EarthmanT) [inline virtual-machine/aws.yaml]: You might have to move `aws_region_name` before `availability_zone`.

File: aws-example-network/README.md (hunk 1 above)
Comment (AdarShaked) [reply aws-example-network/README.md]: its already done in this PR, I will fix the documentation.

**Pull request merged**